docs/guides/connecting_to_your_data/how_to_configure_a_dataconnector_to_introspect_and_partition_a_file_system_or_blob_store.md

```diff
@@ -14,7 +14,7 @@ File-based introspection and partitioning are useful for:
 - Exploring the types, subdirectory location, and filepath naming structures of the files in your dataset, and
 - Organizing the discovered files into <TechnicalTag tag="data_asset" text="Data Assets" /> according to the identified structures.
 
-`Partitioning` enables you to select the desired subsets of your dataset for [Validation](../../reference/validation.md).
+`Partitioning` enables you to select the desired subsets of your dataset for [Validation](../../terms/checkpoint.md).
```

Comment:
```suggestion
`Partitioning` enables you to select the desired subsets of your dataset for [Validation through a Checkpoint](../../terms/checkpoint.md).
```

I think if we're linking to the checkpoint doc then we should make sure to update the link to mention Checkpoints -- otherwise it's a bit non-intuitive.

It would also be reasonable to replace this link with a technical term tag, if this is the first instance where it would appear in the document, or to remove the link entirely if the technical term page is linked previously.  For the sake of consistency: the technical terms "Validate" and "Validation" both redirect to `guides/validation/validate_data_overview` when wrapped in technical term tags -- so we should either redirect to there, change the linked text, or remove the link entirely.